Apsitvarkom.UnitTests/DataAccess/GoogleGeocoderTests.cs

```diff
@@ -1,48 +1,88 @@
 using System.Net;
 using Apsitvarkom.Configuration;
 using Apsitvarkom.DataAccess;
+using Apsitvarkom.Models;
 using Moq;
 using Moq.Protected;
 
 namespace Apsitvarkom.UnitTests.DataAccess;
 
 public class GoogleGeocoderTests
 {
-    private static readonly IApiKeyProvider _apiKeyProvider = new Mock<IApiKeyProvider>().Object;
+    private Mock<IApiKeyProvider> _apiKeyProvider;
+    private Mock<HttpMessageHandler> _handlerMock;
+    private IGeocoder _geocoder;
```

Comment:
Missing `readonly` ?

Reply:
`readonly` properties can only be assigned in constructors and here I am using `SetUp` only.

Reply:
similarly is done in [here](https://github.com/vu-vibedosa/apsitvarkom-be/blob/7f3f5a065c7409df19e1ab147e379d0d4f7ffabf/Apsitvarkom.UnitTests/Models/Mapping/PollutedLocationMappingTests.cs#L12) or [here](https://github.com/vu-vibedosa/apsitvarkom-be/blob/7f3f5a065c7409df19e1ab147e379d0d4f7ffabf/Apsitvarkom.UnitTests/DataAccess/PollutedLocationDTOFileRepositoryTests.cs#L19)

Reply:
ahh I forgot this was not the constructor lol